### DISCUSSION TITLE
緯創前進高雄智慧城市展！大秀移動智能、防疫旅館

### DISCUSSION THREAD

**Discussion by 110409538** [1101-GS4719-C/Python-Community-GS4719-C#517] [科技新知]
實現零接觸旅館服務將能大幅降低防疫旅館工作人員的風險。透過客房床單、被單無接觸盤點及多維度 IoT 裝置，能夠隨時掌握房客住宿期間狀態。

新聞來源:https://finance.technews.tw/2022/03/23/ims-intelligent-mobility-solutions/

**Comment by ThomasMeng0515** [comment]
現在人工智慧蓬勃發展，智慧家庭、智慧城市都變成了炙手可熱研究方向，讓我們的生活能更加便利

**Comment by hapiwangy** [comment]
IOT裝置的發展讓我們在疫情如此嚴重的現在能夠有更多安全上的保障。

**Comment by sean1325418** [comment]
感覺這樣以後服務員都會被機器人給取代了

**Comment by xkllkx** [comment]
看來旅宿業在將來又會有一波失業潮了。

**Comment by PoooYen** [comment]
科技正在逐漸取代我們的生活....

**Comment by cordydeng** [comment]
對於疫情愈發嚴重的現在，感覺這個技術可以有效的阻隔確診者與旅館員工的接觸

**Comment by jcpan821** [comment]
這樣可以有效降低感染風險

**Comment by d26488399** [comment]
很棒的發明，可以讓工作人員的安全更有保障。

**Comment by 11timyyy** [comment]
在防疫當下，希望此科技能造福更多人。

**Comment by Lai27330** [comment]
透過這種AI技術達到零接觸的功能可以保護到更多人免於接觸感染，也能讓照護人員的健康不受威脅。

**Comment by yenwei12138** [comment]
以防疫角度還說很好，但疫情緊張期過後旅宿業人力需求不知回不回地來往日光景
